script.js

```diff
@@ -0,0 +1,88 @@
+/* book template structure */
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+/* function to create a book */
+function addBook(book) {
+  if (book.title !== undefined) {
+    const list = document.querySelector('#book-list');
+    const newRow = document.createElement('tr');
+
+    newRow.innerHTML = `
+    <td>${book.title}</td>
+    <td>${book.author}</td>
+    <button class="remove">Remove</button>
+    <hr>
+    `;
+
+    list.appendChild(newRow);
+  }
+}
```

Comment:
[OPTIONAL] Kindly consider setting a unique id property for each book created. :book: This will make it easier when removing books that have duplicate or same values :+1: